src/test/scala/chiseltest/simulator/VerilatorCirct.scala

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiseltest.simulator
+
+import chiseltest.simulator.jna.JNAUtils
+import chiseltest.utils.CaptureStdout
+import org.scalatest.Tag
+import chiseltest.utils.FlatSpecWithTargetDir
+
+/** To disable tests that require the commercial VCS simulator use the following: `sbt testOnly -- -l RequiresVerilator` */
+object RequiresVerilatorCirct extends Tag("RequiresVerilatorCirct")
+
+class VerilatorCirctBasicCompliance extends BasicCompliance(VerilatorCirctSimulator, RequiresVerilatorCirct)
+class VerilatorCirctStepCompliance extends StepCompliance(VerilatorCirctSimulator, RequiresVerilatorCirct)
+class VerilatorCirctPeekPokeCompliance extends PeekPokeCompliance(VerilatorCirctSimulator, RequiresVerilatorCirct)
+class VerilatorCirctWaveformCompliance extends WaveformCompliance(VerilatorCirctSimulator, RequiresVerilatorCirct)
+class VerilatorCirctCoverageCompliance extends CoverageCompliance(VerilatorCirctSimulator, RequiresVerilatorCirct)
+class VerilatorCirctMemoryCompliance extends MemoryCompliance(VerilatorCirctSimulator, RequiresVerilatorCirct)
+class VerilatorCirctStopAssertAssumeCompliance extends StopAssertAssumeCompliance(VerilatorCirctSimulator, RequiresVerilatorCirct)
```

Comment:
This looks good. However, you will also have to add a task to the CI to actually run these tests.

Reply:
I'm not prepared to add these to CI at this point, installation of firtool is still being refined.

Reply:
OK. Just note that the merge of the PR will be blocked until it has working CI. I am happy to review the draft of course.